redgenes/gene_annotations.py

```diff
@@ -267,11 +316,32 @@ def annotation_pipeline(df, tmpdir, kofamscan_profile, kofamscan_kolist):
                     threshold,
                     score,
                     e_value,
-                    ko_definition)
-                VALUES (?, ?, ?, ?, ?, ?, ?);"""
+                    ko_definition,
+                    run_id)
+                VALUES (?, ?, ?, ?, ?, ?, ?, ?);"""
             TRN.add(sql_ko_info, args_ko_info, many=True)
 
+            barrnap_version = rrna_df["source"].tolist()[0]
+            sql_extract_barrnap_run_info = """
+                SELECT run_id FROM run_info
+                WHERE software = 'barrnap' and version = ? and commands = ?"""
```

Comment:
Part of the motivation for the access object as well is it would reduce code replication :)